examples/flowertune-llm/flowertune_llm/server_app.py

```diff
@@ -0,0 +1,94 @@
+"""flowertune-llm: A Flower / FlowerTune app."""
+
+import os
+from datetime import datetime
+
+from flwr.common import Context, ndarrays_to_parameters
+from flwr.common.config import unflatten_dict
+from flwr.server import ServerApp, ServerAppComponents, ServerConfig
+from flwr.server.strategy import FedAvg
+from omegaconf import DictConfig
+
+from flowertune_llm.client_app import get_parameters, set_parameters
```

Comment:
Let's better move those two functions inside `models.py`. The motivation is to have better isolation between `ServerApp` and `ClientApp`. Currently the `ServerApp` will need to load all the imports defined in `client_app.py`  just to make use of those small functions.